operations/deletion_test.go

```diff
@@ -0,0 +1,48 @@
+package operations
+
+import "testing"
+
+func TestDeletionEquivIfEqualFields(t *testing.T) {
+	del1 := NewDeletion(0)
+	del2 := NewDeletion(0)
+
+	if del1.Equals(del2) == false {
+		t.Errorf("Deletions %v and %v should be considered equal.", del1, del2)
+	}
+}
+
+func TestDeletionEquivIfSameInstance(t *testing.T) {
+	del1 := NewDeletion(0)
+
+	if del1.Equals(del1) == false {
+		t.Errorf("Deletion %v should be considered equal to itself.", del1)
+	}
+}
+
+func TestDeletionNotEquivIfDiffField(t *testing.T) {
+	del1 := NewDeletion(0)
+	del2 := NewDeletion(1)
+
+	if del1.Equals(del2) == true {
+		t.Errorf("Deletions %v and %v should NOT be considered equal.", del1, del2)
+	}
+}
+
+func TestDeletionNotEquivIfDiffFields(t *testing.T) {
```

Comment:
Isn't this test the same as the previous one? I guess we might eventually add more fields to the Deletion type, in which case it might make sense to keep it, but it seems redundant right now. I guess we can leave it though if you think that's best.

Reply:
You're right, this is a copy+paste artefact. I'll get rid of it!